examples/example_metadata.json

```diff
@@ -135,6 +135,6 @@
       }
     },
     "primary_column": "geometry",
-    "version": "1.1.0-dev"
+    "version": "1.1.0"
```

Comment:
@tschaub - I think this is right on this branch. Your [link in the comment](https://github.com/opengeospatial/geoparquet/pull/230#issuecomment-2174005239) looks like it goes to the PR for the projjson schema fix - https://github.com/opengeospatial/geoparquet/commit/a684d7a72f950f290e64f057d47932ae03e457c6. I see it right [on the 1.1 release branch](https://github.com/opengeospatial/geoparquet/blob/e61f1516879c2e7f494cb99cab4492144f0f4d91/examples/example_metadata.json#L138). 

Let me know if I'm missing anything.

Reply:
My bad.  Looks right in this change.

Reply:
No prob! Thanks for checking. A +1 on the release is appreciated :)